tests/functional/test_s3.py

```diff
@@ -409,6 +409,74 @@ def test_client_region_remains_aws_global_for_regional(self):
         self.assertEqual(client.meta.region_name, 'aws-global')
 
 
+class TestS3Copy(BaseS3OperationTest):
+
+    def setUp(self):
```

Comment:
We probably don't need the `setUp` definition here.

Reply:
Good catch, I pulled the customizations out and this got missed in the clean up. Removed in latest revision.